sentinel/src/lib/ids.js

```diff
@@ -99,19 +100,18 @@ const putIdLengthDoc = (db, idLengthDoc) =>
  */
 const findUnusedIds = (db, freshIds) =>
   new Promise((resolve, reject) => {
-    db.medic.view('medic-client', 'contacts_by_reference', {
-      keys: [...freshIds].map(id => [ 'shortcode', id ])
-    }, (err, results) => {
+    const keys = [...freshIds].map(id => [ 'shortcode', id ]);
+    async.parallel([
+      cb => db.medic.view('medic-client', 'contacts_by_reference', { keys: keys }, (err, res) => cb(err, res)),
+      cb => db.medic.view('medic-tombstone', 'contacts_by_reference', { keys: keys }, (err, res) => cb(err, res))
```

Comment:
I think you can replace `, (err, res) => cb(err, res)` with just `, cb` to make this lines easier to read.